agent/tests/test_php_execute.c

```diff
@@ -317,7 +297,7 @@ static void test_stack_depth_after_exception() {
   arg = tlib_php_request_eval_expr("0");
   expr = nr_php_call(NULL, "rethrow", arg);
   tlib_pass_if_int_equal(
-      "PHP stack depth tracking should be 2 after function call", 1,
+      "PHP stack depth tracking should be 2 after function call", 0,
```

Comment:
```suggestion
      "PHP stack depth tracking should be 0 after function call", 0,
```
Does it even make sense to test for stack depth anymore considering your changes in this PR?

Reply:
changed in [bce3e14](https://github.com/newrelic/newrelic-php-agent/pull/767/commits/bce3e1462db26bb5f841542c199dae92ba96a1b3)

I believe that we still want to test and track stack depth.